qubesadmin/tools/qvm_ls.py

```diff
@@ -58,7 +58,7 @@ class Column(object):
 
     def __init__(self, head, attr=None, doc=None):
         self.ls_head = head
-        self.__doc__ = doc if doc is None else qubesadmin.utils.format_doc(doc)
+        self.__doc__ = doc
```

Comment:
It was needed before throwing Admin API in between - there it applied also to all VM properties, where markup is used sometimes. But indeed now it isn't needed anymore.